src/tokenGenerator.js

```diff
@@ -76,22 +78,29 @@ function updateTKK() {
       else {
         let res = await got('https://translate.google.com');
 
-        const code = res.body.match(/TKK=(.*?)\(\)\)'\);/g);
+        const code = res.body.match(/tkk:'\d+.\d+'/g);
```

Comment:
* is `tkk` really become lowercased?
* there is no need in using `g` in regex if we later use only 1st match (`code[0].split...`)
* imo there is point in adding example of response body and highlighting of meaningful part so other become aware of exact thing we are interested in

Reply:
![image](https://user-images.githubusercontent.com/10253982/61628604-3379b280-ac9c-11e9-84f5-8df893e0f430.png)

Just go to and view source of [https://translate.google.com/](https://translate.google.com/)
I am trying to extract value of **tkk**

Reply:
So, the tkk now being lowercase was the issue for us.
And btw, for the regex, you can do a case insensitive match so that it won't matter what case it's in.

Reply:
Yes was lowercased, moved into json, as I can see from code, before it was in other format.